APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,212 @@
+`roles <http://www.astropy.org/team.html>`_.
+
+One additional consequence of this change is that coordinated packages will live
+in the `astropy github organization <https://github.com/astropy>`_ , for which
+the project has management power, while affiliated packages, in general, should
+*not*.  To ease this process, however, any affiliated packages currently in the
+astropy organization can be "legacied-in".  This means that they may remain in
+the github organization even if they are not coordinated packages. Moving
+forward, though, new coordinated packages should be in the astropy organization
+and new  affiliated packages should not be.
+
+Note, though, that affiliated packages are still considered a part of the
+"Astropy Project".  That umbrella term applies to both coordinated and
+affiliated packages, as well as the overall community.
+
+
+A criteria-based package review process
+=======================================
+
+This APE also proposes a second change to the review process, motivated by the
+concerns in the "Motivation" section above about the committee's review
+process. To counter these concerns, prior to this APE the coordination committee
+has informally adopted a checklist-based approach, checking if certain "bullet
+points" are met.  While the formal criteria are listed in relatively vague terms
+on the affiliated package web site, the details of how those criteria are met
+have been fairly consistent using an unwritten but fairly standard list. In
+practice, this has meant that affiliated packages are given "provisional" status
+if some of these items are checked off are met, or "full" if a larger set of
+them are.
+
+This de facto checklist implies that a more transparent and understandable
+policy would be to make this checklist formal and completely public, and use it
+directly as the formal review criteria - i.e., a rubric.  These criteria will be
+laid out explicitly in a document, with various levels defined for each criteria
+(generally corresponding to "good", "acceptable", and "unacceptable"). This
+will make the review process much faster and (eventually) open the review
+process beyond the coordination committee, thereby address the concerns from the
+"Motivation" section.
+
+
+A community review process for affiliated packages
+==================================================
+
+On a longer term horizon, a criteria-based review checklist will allow the
+review process to change to a more "peer-review" model.  That is, instead of
+every package being vetted by the coordination committee, the committee can act
+more as "editors", sometimes offering the review opportunity to other members of
+the Astropy community.  While the details of this process will not be outlined
+here, as they by necessity should be flexible enough to accomodate the
+experiences of the first few reviewers, this APE sets this as the eventual
+*goal* for the affiliated package review process.
```

Comment:
I think it would be useful to already outline this process here. It sounds as if this will essentially follow the format of established journal article peer review with (as the text says) the coordination committee as editors. So, why not include this in the APE? The fact that it might be implemented a little later is fine. In APEs we usually lay out interfaces, data structures etc. that are still going to be implemented at some point in the future. This almost sounds as we we want a second APE for that process. IT would be simpler to include the basics here and leave the rest to the editors.